plugins/wiggle/src/LinearWiggleDisplay/models/model.tsx

```diff
@@ -586,6 +596,25 @@ const stateModelFactory = (
             ),
           )
         },
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        async renderSvg(opts: any) {
```

Comment:
maybe just use `unknown` here?

Reply:
Unknown actually doesn't work, it says unknown not assignable to there. I can use opts: Paramters<typeof superRenderSvg>[0] to get the actual thing expected by superRenderSvg though